core/modules/script.py

```diff
@@ -75,6 +176,8 @@ class Script(Tree):
     leafSkelCls = ScriptLeafSkel
     nodeSkelCls = ScriptNodeSkel
 
+    accessRights = ("add", "edit", "view", "delete", "run")
```

Comment:
Respect PEP-8 conformity, no camel-case.
```suggestion
    access_rights = ("add", "edit", "view", "delete", "run")
```